sky/cli.py

```diff
@@ -1059,17 +1059,21 @@ def cli():
               default=False,
               is_flag=True,
               help='If True, do not actually run the job.')
-@click.option('--detach-setup',
-              '-s',
-              default=False,
-              is_flag=True,
-              help='if True, run setup asynchronously.')
-@click.option('--detach-run',
-              '-d',
-              default=False,
-              is_flag=True,
-              help='If True, run setup first (blocking), '
-              'then detach from the job\'s execution.')
+@click.option(
+    '--detach-setup',
+    '-s',
+    default=False,
+    is_flag=True,
+    help=(
+        'If True, run setup in non-interactive mode where ctrl-c will not '
+        'interrupt the setup process. Useful for long-running setup commands.'))
```

Comment:
We may want to suggest the following so users know how to cancel?
```suggestion
        'If True, run setup in non-interactive mode as part of the job itself. You can safely ctrl-c to detach from logging, and it will not '
        'interrupt the setup process. Setup can be cancelled by canceling the job via `sky cancel`. Useful for long-running setup commands.'))
```
Same elsewhere, if you agree with the wording